portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+async function fetchMessage() {
+    const response = await fetch('/data');
+    const message = await response.text();
+    document.getElementById('fetchmessage-container').innerText = message;
+}
```

Comment:
Please fix "No newline at end of file". Some interesting reading: https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Fixed, thanks!